keyboards/korno65/readme.md

```diff
@@ -0,0 +1,17 @@
+# korno65
+
+Korno65
+
+* Keyboard Maintainer: [Jordan-Ramos](https://github.com/Jordan_Ramos)
+* Hardware Supported: PCB v3
+* Hardware Availability: https://eschit.com/products/gb-korno65s-r2-v3
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader?  Eg, a hardware reset button, etc.

Reply:
added instructions here: https://github.com/qmk/qmk_firmware/pull/13123/commits/f972304a31ecc20cf739c5066b3bcc2f5a7a2a8e